docs/pages/development/installation.md

```diff
@@ -8,31 +8,31 @@ import ConfigurationDiff from '~/components/plugins/ConfigurationDiff';
 import { Tab, Tabs } from '~/components/plugins/Tabs';
 import { Terminal } from '~/ui/components/Snippet';
 
-The installation steps on this page are only required to add the `expo-dev-client` library to a React Native or Bare project. To add a the `expo-dev-client` library to an existing managed project, see our [Getting Started guide](getting-started.md).
-
-If you're just starting your project, you can create a new project from our template with:
+> Note: These docs assume a basic level of familiarity with React-Native. If you are new to React-Native, please see the [React-Native docs](https://reactnative.dev/docs/getting-started).
 
-<Terminal cmd={["$ npx crna -t with-dev-client"]} />
+The installation steps on this page are only required to add the `expo-dev-client` library to a React Native or Bare project. To add a the `expo-dev-client` library to an existing managed project, see our [Getting Started guide](getting-started.md).
```

Comment:
```suggestion
The installation steps on this page are only required to add the `expo-dev-client` library to a React Native or bare project. To add a the `expo-dev-client` library to an existing managed project, see our [Getting Started guide](getting-started.md).
```

i personally prefer "bare" instead of "Bare" - we need to settle which direction to go with this in our writing docs